src/data/set/basic.lean

```diff
@@ -1227,6 +1227,24 @@ ext $ λ s, subset_empty_iff
 @[simp] theorem powerset_univ : 𝒫 (univ : set α) = univ :=
 eq_univ_of_forall subset_univ
 
+/-! ### Sets defined as an if-then-else -/
+
+@[simp] lemma mem_ite_univ_right (p : Prop) [decidable p] (t : set α) (x : α) :
+  x ∈ ite p t set.univ ↔ (p → x ∈ t) :=
+by split_ifs; simp [h]
+
+@[simp] lemma nmem_ite_univ_right (p : Prop) [decidable p] (t : set α) (x : α) :
+  x ∉ ite p t set.univ ↔ (p ∧ x ∉ t) :=
+by split_ifs; simp [h]
+
+@[simp] lemma mem_ite_univ_left (p : Prop) [decidable p] (t : set α) (x : α) :
+  x ∈ ite p set.univ t ↔ (¬ p → x ∈ t) :=
+by split_ifs; simp [h]
+
+@[simp] lemma nmem_ite_univ_left (p : Prop) [decidable p] (t : set α) (x : α) :
+  x ∉ ite p set.univ t ↔ (¬ p ∧ x ∉ t) :=
+by split_ifs; simp [h]
```

Comment:
This can't be `simp` and I'm not sure you want it anyway. Same for `nmem_ite_univ_right`. Also note that the naming convention is `not_mem`, not `nmem`.

Reply:
Is there a clear naming convention? There are some other nmem in that same file. But alright, I'll change it.

Reply:
Searching for `nmem` vs `not_mem` in mathlib gives a clear answer.

Reply:
And in the end I've removed all the not_mem lemmas, since there provable by simp anyway.